React/README.md

```diff
@@ -81,7 +81,7 @@ Forked from the excellent [Airbnb React Style Guide](https://github.com/react),
 
 ## Naming
 
-  - **Extensions**: Use `.jsx` extension for React components.
+  - **Extensions**: Use `.js` extension for React components.
   - **Filename**: Use PascalCase for filenames. E.g., `ReservationCard.jsx`.
```

Comment:
I guess you will need to update this line in order to match the change from above.